server/routes/homeRouter.test.ts

```diff
@@ -0,0 +1,48 @@
+import type { Express } from 'express'
+import request from 'supertest'
+
+import { appWithAllRoutes } from './testutils/appSetup'
+
+let app: Express
+
+beforeEach(() => {
+  jest.clearAllMocks()
```

Comment:
i suspect you don't need this, but no harm in it i guess

Reply:
Yeah I was copying this from the other test where you check the analytics tiles. Originally I was tempted to move those tests into this file (they mock S3 things), but then I decided to not do so and keep this changeset focused on the table stuff.